From 94d4e04bd52f265f8119c77e66c672096fed88f9 Mon Sep 17 00:00:00 2001
From: Jose Dapena Paz <jdapena@igalia.com>
Date: Tue, 6 Aug 2024 13:46:17 +0200
Subject: libstdc++: avoid requiring type declaration for unique_ptr members

Assigning nullptr to std::unique_ptr requires the destructor
declaration for the type in libstdc++ implementation.

Bug: 41455655
Change-Id: I55f0f963b7783f8a31939bef04d7b00ce9c13ef3
---
 chrome/browser/ui/browser_actions.h | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/chrome/browser/ui/browser_actions.h b/chrome/browser/ui/browser_actions.h
index 6d31e2327ccc3..3b4ce4dae93fe 100644
--- a/chrome/browser/ui/browser_actions.h
+++ b/chrome/browser/ui/browser_actions.h
@@ -41,9 +41,8 @@ class BrowserActions {
 
   raw_ptr<actions::ActionItem> root_action_item_ = nullptr;
   std::unique_ptr<TranslateBrowserActionListener>
-      translate_browser_action_listener_ = nullptr;
-  std::unique_ptr<BrowserActionPrefsListener> browser_action_prefs_listener_ =
-      nullptr;
+      translate_browser_action_listener_;
+  std::unique_ptr<BrowserActionPrefsListener> browser_action_prefs_listener_;
   const raw_ref<Browser> browser_;
 };
 
-- 
2.43.0

